.github/workflows/build.yml

```diff
@@ -45,10 +73,50 @@ jobs:
             ${{ runner.os }}-sbt-${{ hashFiles('project/build.properties') }}-${{ hashFiles('project/plugins.sbt') }}
           restore-keys: ${{ runner.os }}-sbt-
           
-      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }} / ${{matrix.platform }}
+      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }}, ScalaJS ${{ matrix.scalajs }}
+        run: |
+          ./.github/scripts/build
+        env:
+          SCALA_VERSION: ${{ matrix.scala }}
+          SCALAJS_VERSION: ${{ matrix.scalaJS }}
+          SBT_COMMAND: ci-js
+  
+  mima:
+    name: Mima binary compatibility test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        java: [ 8 ]
+        scala: [ 2.12.11 ]
+        scalajs: [ 0.6.32 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: "adopt@1.${{ matrix.java }}"
+     
+      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }}, ScalaJS ${{ matrix.scalajs }}
         run: |
           ./.github/scripts/build
         env:
           SCALA_VERSION: ${{ matrix.scala }}
           SCALAJS_VERSION: ${{ matrix.scalaJS }}
-          PLATFORM: ${{ matrix.platform }}
+          SBT_COMMAND: ci-jvm-mima
```

Comment:
We could change `ci-jvm-mima` to not run tests, it runs them twice for this version, right?

Reply:
did not know that mima task was running test too, therefore yes, tests for 2.12.11 are being executed twice.

Reply:
It's defined in build.sbt: https://github.com/monix/monix/blob/master/build.sbt#L22

Reply:
In order to make this easier, could you tick the tests that needs to be executed and left empty the not required ones?

JVM
- [ ] 8, 2.11
- [ ] 8, 2.11 (by mima check)
- [ ] 8, 2.12
- [ ] 8, 2.13
- [ ] 11, 2.11
- [ ] 11, 2.12
- [ ] 11, 2.13

JS
- [ ] 8, 2.11, 0.6.32
- [ ] 8, 2.12, 0.6.32
- [ ] 8, 2.13, 0.6.32
- [ ] 8, 2.11, 1.0.0 
- [ ] 8, 2.12, 1.0.0 
- [ ] 8, 2.13, 1.0.0
- [ ] 11, 2.11, 0.6.32
- [ ] 11, 2.12, 0.6.32
- [ ] 11, 2.13, 0.6.32
- [ ] 11, 2.11, 1.0.0
- [ ] 11, 2.12, 1.0.0
- [ ] 11, 2.13, 1.0.0

Reply:
Sure, I ticked few boxes - I feel like it's enough to focus on jdk8 + sanity check on one jdk11 combination. I don't know if it matters in ScalaJS

I checked ` 8, 2.11 (by mima check)` but we can do `8, 2.11` and mima check separately if it's easier to configure